integration-tests/smoke/vrfv2plus_test.go

```diff
@@ -1424,6 +1424,153 @@ func TestVRFV2PlusWithBHS(t *testing.T) {
 	})
 }
 
+func TestVRFV2PlusWithBHF(t *testing.T) {
+	t.Parallel()
+	var (
+		env                          *test_env.CLClusterTestEnv
+		vrfContracts                 *vrfcommon.VRFContracts
+		subIDsForCancellingAfterTest []*big.Int
+		defaultWalletAddress         string
+		vrfKey                       *vrfcommon.VRFKeyData
+		nodeTypeToNodeMap            map[vrfcommon.VRFNodeType]*vrfcommon.VRFNode
+	)
+	l := logging.GetTestLogger(t)
+
+	config, err := tc.GetConfig("Smoke", tc.VRFv2Plus)
+	require.NoError(t, err, "Error getting config")
+	vrfv2PlusConfig := config.VRFv2Plus
+
+	cleanupFn := func() {
+		if env.EVMClient.NetworkSimulated() {
+			l.Info().
+				Str("Network Name", env.EVMClient.GetNetworkName()).
+				Msg("Network is a simulated network. Skipping fund return for Coordinator Subscriptions.")
+		} else {
+			if *vrfv2PlusConfig.General.CancelSubsAfterTestRun {
+				//cancel subs and return funds to sub owner
+				vrfv2plus.CancelSubsAndReturnFunds(testcontext.Get(t), vrfContracts, defaultWalletAddress, subIDsForCancellingAfterTest, l)
+			}
+		}
+		if !*vrfv2PlusConfig.General.UseExistingEnv {
+			if err := env.Cleanup(); err != nil {
+				l.Error().Err(err).Msg("Error cleaning up test environment")
+			}
+		}
+	}
+
+	// BHF job config
+	config.VRFv2Plus.General.BHFJobWaitBlocks = ptr.Ptr(260)
```

Comment:
But we can use 256 blocks, why use a bit more?

Reply:
In [PROD](https://github.com/smartcontractkit/vrf-job-specs/blob/main/bhf/arbitrum-mainnet/primary.toml) it is set to 256 and it works fine

Reply:
Yes, that is true. Since, this is QA, I'm testing a more general scenario of BHF